webapp/app/data_access/form_data_controller.py

```diff
@@ -0,0 +1,35 @@
+import redis
+import redislite
+from app.config import Config
+from pydantic import MissingError
+
+redis_url = Config.SESSION_DATA_STORAGE_URL
+if redis_url == 'None':
+    r = redislite.StrictRedis()
+else:
+    r = redis.Redis.from_url(Config.SESSION_DATA_STORAGE_URL)
+
+
+def save(key: str, value) -> bool:
+    """Saves/updates a key/value pair that expires given the config TTL.
+    :param str key: the key value including the user identifier, so all keys are unique.
+    :param value: the value corresponding to the key.
+    :return: true or false depending on the successfulness of the operation.
+    :rtype: bool
+    """
+    ttl_seconds = Config.SESSION_DATA_REDIS_TTL_HOURS*3600
+    return r.setex(key, ttl_seconds, value)
+
+
+def get(key: str) -> str:
```

Comment:
Personally I would call these two methods `get_from_redis` and `save_to_redis` to make them more explicit. wdyt?

Reply:
Jep, is more readable :)